components/authz-service/storage/v2/postgres/postgres.go

```diff
@@ -865,10 +866,41 @@ func (p *pg) CreateProject(ctx context.Context, project *v2.Project) (*v2.Projec
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
-	if err := p.insertProjectWithQuerier(ctx, project, p.db); err != nil {
+	tx, err := p.db.BeginTx(ctx, nil)
+	if err != nil {
+		return nil, p.processError(err)
+	}
+
+	rows, err := tx.QueryContext(ctx, `SELECT query_projects($1)`, pq.Array([]string{}))
```

Comment:
This may be able to be simplified further but it will save some calculation:
`WITH t as (SELECT query_projects()) SELECT COUNT(*) FROM t;`

Reply:
The max `n` in this case is 6, so I think we are ok on calculation time ;)

Reply:
Ah, yeah. I really meant by that lines of code. I am always looking for how to reduce the lines of code.